File: app/src/main/java/org/kindone/willingtodo/persistence/ConfigPersistenceProvider.java
package org.kindone.willingtodo.persistence;

import org.kindone.willingtodo.data.TaskContext;

import java.util.List;

/**
 * Created by kindone on 2016. 12. 22..
 */

public interface ConfigPersistenceProvider {

    int getTabIndex();

    void saveTabIndex(int index);

    int getVersion();
}

File: app/src/main/java/org/kindone/willingtodo/persistence/ConfigPersistenceProvider.kt
package org.kindone.willingtodo.persistence

import org.kindone.willingtodo.data.TaskContext

/**
 * Created by kindone on 2016. 12. 22..
 */

interface ConfigPersistenceProvider {

    val tabIndex: Int

    fun saveTabIndex(index: Int)

    val version: Int
}
